src/modules/project/sagas.ts

```diff
@@ -116,8 +116,14 @@ function* handleDuplicateProject(action: DuplicateProjectAction) {
 
   const scene = yield getSceneByProjectId(project.id)
 
+  let thumbnail = project.thumbnail
+
+  if (thumbnail && !thumbnail.startsWith('data:image/png;')) {
```

Comment:
should we use `startsWith('http')` to keep it consitent? (https://github.com/decentraland/builder/blob/master/src/components/ProjectCard/ProjectCard.tsx#L72) 

Also if we are going to `fetch` it should be an http/s endpoint

Reply:
doh, yep